### DISCUSSION TITLE
View build output

### DISCUSSION THREAD

**Discussion by eggers** [vercel/vercel#5496] [Help] [answered]
Our build process runs some analysis and saves the output to a directory. Is there a way of accessing the build directory and downloading files from there?

**Comment by styfle** [comment, marked as answer]
Your build will typically output files to the `public` directory and these files can be download by visiting your deployment URL.

You can configure the [Output Directory](https://vercel.com/docs/build-step#output-directory) if you wish to use a different directory.
